### PR TITLE
add blog: wechaty-plugin-xyao [give bot ability to handle instruction distributed]

### PR DESCRIPTION
**Pull request by watertao**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.
e.g.

``` md
---
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [x] Add `<!--more-->` section to just show the abstract in blog homepage.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.github.io#contribute-guideline)

### CODE REVIEW

**Comment by lijiarui** [comment]
Thanks for the blog!

Since we change the template or wechaty blog, we have a break change:

## Before
You should add your profile at `author.yml`

## After

### 1. Create a file at `_developers` named as `yourname.md` 

 `yourname` should be the same as the author name you put in the blog:
```
---
title: XX
date: XX
author: yourname
categories: XX
tags:
  - XX
image: XX
---
```
### 2. The file should begin with:
```
---
name: Jiarui LI (李佳芮)
site: https://lijiarui.github.io
bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
avatar: /assets/developers/lijiarui/avatar.png
email: rui@juzi.bot
twitter: https://weibo.com/u/2175505900
---
```

### 3. Example:

- lijiarui.md: https://github.com/wechaty/wechaty.github.io/blob/master/_developers/lijiarui.md
- It will shown like this on wechaty blog: https://wechaty.github.io/developers/lijiarui/

Looking forward to your profile file at the `_developers` folder!

**Comment by watertao** [comment]
@lijiarui Thanks for review, my profile has been committed.

File: _posts/2020-08-01-wechaty-distribute-instruction-handler.md [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+节点的处理能力，也限制了能力的扩展。将微信通讯与业务解耦，业务和业务独立部署，能够有效缓解这些问题。
+2. 独立出的业务处理模块可以采用任意适合该领域业务的语言（目前提供了一个基于 java springboot 的 brain 模块开发框架）。
+3. 由于采用了基于 pub/sub 的消息中间件作为机器人和 brain 的通讯，因此即使两者之间由于 NAT 无法提供基于固定公网 IP 的 RPC 服务，也可以通过这种方式
+打通交互。
+
+<!--more-->
+
+## demo
+
+![sample](../assets/2020/wechaty-plugin-xyao/interaction-sample.png)
```
Comment (lijiarui) [inline _posts/2020-08-01-wechaty-distribute-instruction-handler.md]: no need to add `..`

File: _posts/2020-08-01-wechaty-distribute-instruction-handler.md [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+1. `masters`: 机器人的主人账号(注意是contact.id)，有部分指令必须是主人发起的才会执行
+1. `brains_cli`: brain 列表
+1. `brains_ai`: 未被识别为指令的的消息会交给这个 brain 模块
+1. `log_appender`: `dateFile` 输出到指定文件，按日期分割； `stdout` 输出到控制台
+1. `log_file`: 日志文件路径
+1. `log_level`: 日志级别
+
+## 部署架构参考
+
+![architecture](../assets/2020/wechaty-plugin-xyao/arc.png)
```
Comment (lijiarui) [inline _posts/2020-08-01-wechaty-distribute-instruction-handler.md]: no need to add `..`

**Review by lijiarui — CHANGES_REQUESTED** [review]
Good job!

Please change as I request.

**Comment by watertao** [comment]
@lijiarui image link has been corrected

File: _data/authors.yml [hunk 3]
```diff
@@ -971,3 +971,15 @@ csjuXYZ:
     - label: "Email"
       icon: "fas fa-fw fa-envelope-square"
       url: "hoc0415@foxmail.com"
+
```
Comment (lijiarui) [inline _data/authors.yml]: Since we have the profile markdown file, so you should remove this introduction commit from authors.yml.

**Review by lijiarui — CHANGES_REQUESTED** [review]

**Comment by watertao** [comment]
fail to exclude authors.yml from this pr, so close it and recreate a new pull request

**Pull request closed without merging**